### PR TITLE
chore: common configs support

### PR DESCRIPTION
**Pull request by CL-SHLOMIKONCHA**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✔/✖                                                                        |
| New feature     | ✔/✖                                                                        |
| Breaking change | ✔/✖                                                                        |
| Deprecations    | ✔/✖                                                                        |
| Documentation   | ✔/✖                                                                        |
| Tests added     | ✔/✖                                                                        |
| Chore            | ✔/✖                                                                       |

Related issues: #XXX , #XXX ...
Closes #XXX ...

Further  information:

### CODE REVIEW

**Review by asafMasa — APPROVED** [review]

File: helm/local.yaml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+global:
+  dockerRegistryUrl: acrarolibotnonprod.azurecr.io
+  imagePullSecretName: acr-registry
+  flavor: openshift
+  tracing: {}
+  metrics: {}
+  environment: {}
+  db:
+    host: "10.8.1.4"
```
Comment (yossiz16) [inline helm/local.yaml:9]: please remove

File: helm/local.yaml [hunk 2]
```diff
@@ -0,0 +1,80 @@
+global:
+  dockerRegistryUrl: acrarolibotnonprod.azurecr.io
+  imagePullSecretName: acr-registry
```
Comment (yossiz16) [inline helm/local.yaml:3]: remove

File: helm/local.yaml [hunk 3]
```diff
@@ -0,0 +1,80 @@
+global:
```
Comment (yossiz16) [inline helm/local.yaml:1]: remove

**Review by yossiz16 — CHANGES_REQUESTED** [review]
please remove actuall values

**Review by Yossizz — APPROVED** [review]

**Pull request merged**